geoutils/georaster.py

```diff
@@ -851,17 +854,29 @@ def intersection(self, rst):
         else:
             return extent
 
-    def show(self, band=None, **kwargs):
+    def show(self, band=None, cmap=None, vmin=None, vmax=None, cb_title=None,
+             no_cb=False, ax=None, **kwargs):
```

Comment:
Does it have a specific reason that it's ckaaed `*no*_cb`? I would call it `add_colorbar` or so...that avoids confusion in my head ;-)

Reply:
Mmmh, good question. I think it's mostly because I;m used to run it from command line with argparse. It's easier to make cbar the default, and add a -nocb option to remove it if not wanted.
What do you think? You can look at geoviewer.py how that would translate.

Reply:
Well, if I get it correctly it's about changing the argument name and default in this line, right? (and then of course the subsequent uses) : https://github.com/adehecq/GeoUtils/blob/084b0a263283a9830917e65cd11bed4be8ab5fb6/geoutils/geoviewer.py#L34

Reply:
Just replaced it in commit 36ab162 !

Reply:
Cool thanks!!